### PR TITLE
August Review of PAM Playbook

### PR DESCRIPTION
**Pull request by idmken**

### CODE REVIEW

File: _playbooks/playbook-pam.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+
+There are three main use cases to identify a privileged account or user.
+1. Accounts used to manage IT infrastructure, resources of high value assets (HVA), and core systems (e.g. perform maintenance activities on human resource applications or databases).
+2. Help desk personnel with escalated privilege to perform security-relevant processes on endpoints (e.g. install software on user laptops or change endpoint configuration settings).
+3. Managers that approve access or accounts (e.g. a manager approving access requests to a risk reporting application).
```
Comment (idmken) [inline _playbooks/playbook-pam.md:56]: Change this to just a manager approving or recertifying access to an application.

File: _playbooks/playbook-pam.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+1. These employees and contractors have the ability to jeopardize sensitive information or infrastructure, whether knowingly or unknowingly.
+2. Privileged users have the potential to compromise all three core elements of information security: availability, confidentiality, and integrity.
+
+Agencies can use this playbook to help plan and implement a logical privileged user program following government-wide best practices. This playbook includes a four step process aligned with the [Federal Identity, Credential, and Access Management (ICAM) Architecture](https://playbooks.idmanagement.gov/arch/){:target="_blank"}{:rel="noopener noreferrer"} and is designed for **insider threat, ICAM, and risk management professionals interested in identifying best practices for mitigation privileged user risk** for new or existing privileged user programs. Agencies are encouraged to tailor this playbook to fit their unique organizational structure, mission needs, and requirements. Other IT program participants, including cybersecurity program managers, may find value in incorporating this playbook approach in their planning.
```
Comment (idmken) [inline _playbooks/playbook-pam.md:62]: We originally called this the privileged access management playbook, but I changed it to privileged user playbook. For simplicity that this playbook talks about more than access and didn't want to use any vendor specific terms.

File: _playbooks/playbook-pam.md [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+- **[Insider Threat](https://csrc.nist.gov/glossary/term/insider_threat){:target="_blank"}{:rel="noopener noreferrer"}** is the potential for an insider will use her/his authorized access, wittingly or unwittingly, to do harm to the security of United States. This threat can include damage to the United States through espionage, terrorism, unauthorized disclosure of national security information, or through the loss or degradation of departmental resources or capabilities.
+- **[Functional Privileged User](https://playbooks.idmanagement.gov/docs/playbook-dira.pdf){:target="_blank"}{:rel="noopener noreferrer"}** can access information resources provided by the application and approval workflows such as approving access requests.
+- **[Privileged Account](https://csrc.nist.gov/glossary/term/privileged_account){:target="_blank"}{:rel="noopener noreferrer"}** is a system account used by a privileged user. A privileged account can belong to a single endpoint, domain, database, or application.
+- **Privileged Compromise** is either an adverse action of a privileged user or account through either an insider threat or an account compromise.
```
Comment (idmken) [inline _playbooks/playbook-pam.md:70]: I created this term for the playbook to denote either a compromise from an account (external threat) or an insider (internal threat). We have it in the disclaimer that it isn't an authoritative terms.

File: _playbooks/playbook-pam.md [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+
+# Protect Federal Identities and Logical Assets
+Government employees and contractors need elevated access to perform necessary administrative and security functions, yet this carries an inherent risk of insider threat or account compromise. As a result, agencies should implement privilege user controls that mitigate this risk without impeding their ability to carry out assigned job duties. In creating a secure physical and virtual workplace for privileged users, agencies align efforts with the [FICAM Architecture](https://playbooks.idmanagement.gov/arch/){:target="_blank"}{:rel="noopener noreferrer"}.
```
Comment (idmken) [inline _playbooks/playbook-pam.md:78]: I got stuck saying insider threat or account compromise which which why I went with privileged compromise.

File: _playbooks/playbook-pam.md [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+
+This playbook provides informative guidance to manage federal employee and contractor privileged users by mitigating the inherent risks associated with this population. An agency achieves this outcome through the use of ICAM services. The following are the four primary steps to establish or enhance an agency's privileged user program.
+1. [Define and Identify](#2-define-and-identify) privileged roles as those accounts that have elevated access to an agency's resources. Privileged accounts can be anywhere across a network or system and should be tracked and audited on a regular basis.
```
Comment (idmken) [inline _playbooks/playbook-pam.md:81]: Need to check that we cover all the sections within each step in this part.

File: _playbooks/playbook-pam.md [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+3. [Integrate into ICAM](#3-integrate-into-icam) for a holistic management approach. Implementing ICAM best practices as well as additional countermeasures to prevent privileged user and account misuse, abuse, or compromise can provide comprehensive, integrated protection of agency resources.
+4. [Prioritize and Execute](#4-prioritize-and-execute) on your plan. Like most IT initiatives, this is a journey. Threats change. Based on your agency risk, this may require a multi-year plan to add capabilities and services. An agency should leverage existing efforts to achieve information security goals by directing or tailoring playbook steps based on its resources, environment, mission, business needs, and privileged user population.
+
+{% include alert-info.html heading="Is this Privileged Access Management or Account Security" content="Different vendors may use a different terms for their product. Some vendors may use the term Privileged Access or Account Management (PAM), Privileged Identity Management (PIM), Privileged Security, or something in-between. For the intent of this playbook, the agnostic privileged user is used to encompass all service areas of ICAM." %}
```
Comment (idmken) [inline _playbooks/playbook-pam.md:86]: Clean this up, but you get the idea.

File: _playbooks/playbook-pam.md [hunk 7]
```diff
@@ -0,0 +1,367 @@
...
+
+[![An end user account can be compromised to esclate privileges in a domain takeover]({{site.baseurl}}/assets/playbooks/pam-journey-map.png)]({{site.baseurl}}/assets/playbooks/pam-journey-map.png){:target="_blank"}{:rel="noopener noreferrer"}
```
Comment (idmken) [inline _playbooks/playbook-pam.md:88]: Just noticed that some diagrams don't have any text to explain them.

File: _playbooks/playbook-pam.md [hunk 8]
```diff
@@ -0,0 +1,367 @@
+---
```
Comment (idmken) [inline _playbooks/playbook-pam.md:1]: @JBPayne007 here's the doc to review and add comments.

File: _playbooks/playbook-pam.md [hunk 9]
```diff
@@ -0,0 +1,367 @@
...
+
+{% include alert-info.html heading="Why Additional Controls?" content="Most attacks start by compromising lower level accounts. Through network discovery, an attacker can find an orphaned privileged account to escalate their privileges to access applications, data, and compromise entire agency networks or data sets." %} 
+
+The Privileged User Program should document each step either manually or automated. This step is the basis to maintain and keep accountability for an agency's privileged user population.
+
+[![Privileged accounts can be found on Windows, Unix/Linux, Applications/Cloud, DevOps, and Machine Identities]({{site.baseurl}}/assets/playbooks/pam-identify.png)]({{site.baseurl}}/assets/playbooks/pam-identify.png){:target="_blank"}{:rel="noopener noreferrer"}
+
+## Step 3. Integrate into ICAM
```
Comment (idmken) [inline _playbooks/playbook-pam.md:207]: Include an architecture diagram.

File: _playbooks/playbook-pam.md (hunk 2 above)
Comment (JBPayne007) [reply _playbooks/playbook-pam.md:62]: Ok, But I don't think that privileged access management is a vendor specific term. It is a Cyber security industry standard term but not used as a specific product or vendor name.

File: _playbooks/playbook-pam.md (hunk 1 above)
Comment (JBPayne007) [reply _playbooks/playbook-pam.md:56]: The access / account approval process includes approval by the Information Systems Security Manager (ISSM) as well as Physical Security Manager for clearance level verification in the DoD.

File: _playbooks/playbook-pam.md (hunk 3 above)
Comment (JBPayne007) [reply _playbooks/playbook-pam.md:70]: Sounds good.

File: _playbooks/playbook-pam.md (hunk 4 above)
Comment (JBPayne007) [reply _playbooks/playbook-pam.md:78]: Sure, privileged account compromise.

File: _playbooks/playbook-pam.md (hunk 6 above)
Comment (JBPayne007) [reply _playbooks/playbook-pam.md:86]: This section could get tricky as every agency has their own requirements and software applications that they use to track these privileged user access requirements. Such as different training requirements for each system or software title, clearance level requirements, and Information Assurance Technical Level (IAT Level) requirements from DoD 8570 m1 for DoD.

File: _playbooks/playbook-pam.md [hunk 10]
```diff
@@ -0,0 +1,367 @@
...
+5. Data - The data includes controls on data sharing, data classification and tagging, preventing data loss, and potentially applying data retention policy. 
+
+[![The five zero trust elements include user, device, network, application, and data]({{site.baseurl}}/assets/playbooks/pam-zt.png)]({{site.baseurl}}/assets/playbooks/pam-zt.png){:target="_blank"}{:rel="noopener noreferrer"}
+
+## Step 2. Define and Identify
+An agency is responsible for managing the privileges of all users. To execute assigned duties, a subset of an agency’s user population may be granted elevated access. The individuals entrusted with elevated access constitute an agency’s privileged user population.
+
+### Define a Privileged User
+An agency’s privileged user population can include a range of accounts with elevated access. Given the broad-reaching nature of an organization’s privileged user population, it is important that an agency understand the user roles, groups, and accounts that constitute its privileged user population. Some common examples include:
+- domain administrator
```
Comment (idmken) [inline _playbooks/playbook-pam.md:171]: Include Linux/Unix root, Oracle SYS, Cisco Enable

Comment (idmken) [reply _playbooks/playbook-pam.md:171]: Also include Windows service accounts, hard-coded SSH keys, embedded credentials in code.

Comment (idmken) [reply _playbooks/playbook-pam.md:171]: Emergency or break-glass accounts.

File: _playbooks/playbook-pam.md [hunk 11]
```diff
@@ -0,0 +1,367 @@
...
+
+4. [NIST Special Publication 1800-18 - Privileged Account Management for the Financial Services Sector](https://www.nccoe.nist.gov/projects/use-cases/privileged-account-management){:target="_blank"}{:rel="noopener noreferrer"}
+    - While this special publication is focused on the financial services industry it contains agnostic implementation best practices.
+
+5. [Federal Identity, Credentials, and Access Management (FICAM) Architecture - Access Management](https://playbooks.idmanagement.gov/arch/access/){:target="_blank"}{:rel="noopener noreferrer"}
+     - The FICAM Architecture is a framework for an agency to use in ICAM program and solution roadmap planning. Privileged Access Management is identified as a distinct service within the Access Management portion of the ICAM services framework.
+
+6. [Common Sense Guide to Mitigating Insider Threats (6th Edition), February 2019](https://resources.sei.cmu.edu/library/asset-view.cfm?assetid=540644){:target="_blank"}{:rel="noopener noreferrer"}
```
Comment (idmken) [inline _playbooks/playbook-pam.md:316]: Should add HVA memo - https://www.whitehouse.gov/sites/whitehouse.gov/files/omb/memoranda/2017/m-17-09.pdf

NIST IR 7966 ssh - https://csrc.nist.gov/publications/detail/nistir/7966/final

**Review by JBPayne007 — APPROVED** [review]
Looks good.

**Pull request merged**